pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-pdfs.conf

```diff
@@ -14,6 +14,9 @@
               "type": "ACTIVITY",
               "activityCode": "CONSENT",
               "versionTag": "v2"
+            },
+            {
```

Comment:
Oh wow--so the expressions we put into the pdf templates themselves are restricted here, such that a reference in the template can be silently ignored if the appropriate `source` isn't included in this array?

Reply:
There is an error message in the logs that says the mail address was not found. But it is not found because we never tried to look for it in the first place.